docs/resources/modelarts_resource_pool.md

```diff
@@ -93,5 +238,23 @@ This resource provides the following timeouts configuration options:
 The ModelArts resource pool can be imported using the `id`, e.g.
 
 ```bash
-$ terraform import huaweicloud_modelarts_resource_pool.test 0ce123456a00f2591fabc00385ff1234
+$ terraform import huaweicloud_modelarts_resource_pool.test <id>
```

Comment:
actually, the resource ID equals name?

Reply:
it is not exactly correct, the name of the metadata is consists of the name and the project ID